README.md

```diff
@@ -1,65 +1,73 @@
+<h1 align="center">
+Ribir - Non-invasive Declarative GUI Framework
+</h1>
+<p align="center"><a href="https://ribir.org" target="_blank" rel="noopener noreferrer">
+<img src="website/static/img/logo.svg" alt="Ribir-logo"  width = "128px">
+</a></p>
+
+<p align="center">
+Use Rust to build multi-platform applications from a single codebase.
+</p>
 <div align="center">
-<img src="website/static/img/logo-animation.gif" width="480px" />
-
-<!-- # Ribir -->
-
-Ribir is a framework for building modern native/wasm cross-platform user interface applications.
 
 [![CI](https://github.com/RibirX/Ribir/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/RibirX/Ribir/actions/workflows/ci.yml)
 [![codecov](https://codecov.io/gh/RibirX/Ribir/branch/master/graph/badge.svg)](https://codecov.io/gh/RibirX/ribir)
-[![License](https://img.shields.io/badge/license-MIT-informational)](https://github.com/RibirX/ribir/blob/master/LICENSE)
+[![Crates.io](https://img.shields.io/crates/v/ribir)](https://crates.io/crates/ribir)
+[![Crates.io](https://img.shields.io/crates/d/ribir)](https://crates.io/crates/ribir)
 
-[Documents] | [Examples]
+[Website] | [Documents] | [Examples] | [Changelog]
 
+<!--todo: a demos picture here-->
 </div>
 
+## What's Ribir?
 
-## Principles
-
-- **Non-injection and Non-invasive**: Ribir interacts with the APIs of your data structure, and does not require you to do any pre-design for the user interface. Developers can focus on designing the data struct, logic and APIs. Ribir will neither break your existing logic nor require injecting any of its own objects.
-
-- **Declarative**: The user interface is the description of data, the description will automatically react to the data modification. Once you describe your data, the description will keep describing your data.
-
-- **Purely composed**: Ribir creates UI via widgets. There is not a base-object, even if the built-in fields are provided in a composed way. For example,  only if you use `margin` field, the `Margin` widget will be composed, if you do not use it, you don't pay any overhead for `Margin`. "Only pay for what you need" is an important guideline for Ribir.
+Ribir is a Rust GUI framework to help you build beautiful, natively compiled, multi-platform applications from a single codebase.
 
+It's trying a new way to build UI directly based on the API of your data, and every modification of data will drive the UI update point-to-point. In this way, you can focus on designing the data structure and API of your application and then describe the UI of your data without invasive its logic.  
```

Comment:
A new way to build UI directly based on the API of your data, and every modification of data will drive the UI update point-to-point. You can focus on designing the data structure and API of your application, and then describe the UI of your data without invasive its logic.